server/models/remote_provider.go

```diff
@@ -111,7 +111,10 @@ func (l *RemoteProvider) loadCapabilities(token string) {
 		// Proceed to make a request with the token
 		resp, err = l.DoRequest(req, token)
 	}
-
+	if resp == nil {
+		logrus.Errorf("Could not reach remote provider %s", err)
```

Comment:
```suggestion
		logrus.Errorf(ErrUnreachableRemoteProvider(err))
```

Reply:
Thanks @theBeginner86 for the suggestion